gc/base/EnvironmentBase.hpp

```diff
@@ -76,13 +76,13 @@ typedef enum {
 class MM_EnvironmentBase : public MM_BaseVirtual
 {
 private:
-#if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS) && !defined(OMR_OVERRIDE_COMPRESS_OBJECT_REFERENCES)
-	bool const _compressObjectReferences;
-#endif /* defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS) && !defined(OMR_OVERRIDE_COMPRESS_OBJECT_REFERENCES) */
 	uintptr_t _workerID;
 	uintptr_t _environmentId;
 
 protected:
+#if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS)
+	bool const _compressObjectReferences;
```

Comment:
Glad to see you made the field `const`!

Reply:
Not that it matters terribly, but why did you move the field? (The change would have been smaller if the field order didn't change.)

Reply:
I moved it so it could be next to the other protected fields, but I can add another `protected` label above the existing `private` section so that the field order doesn't change

Reply:
I don't think the GC guys will want that - single areas for each range of visibility is the norm everywhere.